calico_mesos/calico_mesos.py

```diff
@@ -71,6 +72,19 @@ def calico_mesos():
     except KeyError:
         raise IsolatorException(ERROR_MISSING_ARGS)
 
+    # Unfortunately, netmodules passes in labels poorly, so move them
+    # around to match the agreed upon design.
+    if args.has_key('labels'):
+        # copy labels so we're not modifiying the original as we iterate
+        _labels = copy.deepcopy(args['labels'])
+        args['labels'] = {}
+        for label in _labels:
+            # Move the key and value into the labels dictionary
+            key = label['key']
```

Comment:
Yeah I figured that might be the case but wasn't sure.  Maybe worth a mention so I don't ask again next time I review this :-)